src/msdo-client.ts

```diff
@@ -127,7 +127,15 @@ export async function run(inputArgs: string[], successfulExitCodes: number[] = n
         // Write it as an environment variable for follow up tasks to consume
         tl.setVariable('MSDO_SARIF_FILE', sarifFile);
 
-        tool.arg('--export-breaking-results-to-file');
+        if (common.isVersionGreaterThanOrEqualTo(process.env.MSDO_INSTALLEDVERSION, '0.183.0')) {

```

Comment:
>if (common.isVersionGreaterThanOrEqualTo(process.env.MSDO_INSTALLEDVERSION, '0.183.0')) [](http://example.com/codeflow?start=8&length=87)

Can we rollback external CLI releases by delisting? Just wondering if we have to change this code in that case

Reply:
We can :)

This uses the javascript nuget installed which sets `MSDO_INSTALLEDVERSION` to the explicit version discovered via the NuGet REST API.